src/markdown-pages/docs/cli-command-reference/cli-command-reference.md

```diff
@@ -26,19 +26,14 @@ The following versions of the Galasa CLI tool are available to download for diff
 
 ## Getting help
 
-Use the following commands to get more information about the command and command options, including default values.
+Use the following command to get more information about the command and command options, including default values.
 
-On Mac and Unix:
 
 ```
 galasactl -h, --help 
 ```
 
-On Windows:
-
-```
-galasactl.exe -h, --help 
-```
+Remember to specify `galasactl.exe` on your PATH variable if you are using Windows.
```

Comment:
PATH holds a folder into which the executable program is placed. So putting galasactl.exe on your PATH variable separately shouldn't be needed, so long as the folder it sits in is explicitly named on your PATH.

PATH is a search-path, which the command shell uses to look for programs you are trying to invoke. Normally they require an exact match to what you type, but Windows can ignore the .exe extension.

So perhaps this should be :
>"Remember to make sure `galasactl.exe` is on your PATH if you are using windows.

Reply:
updated